package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "sfcc-devtools",
+  "version": "1.0.0",
+  "description": "Support Browser Interaction with VS Code, Eclipse, and SFCC Business Manager via DevTools Web Inspector.",
+  "author": "Red van Workshop",
+  "license": "MIT",
```

Comment:
Consider adding an engines property to specify a minimum node version.  I was not able to get npm test to work when my node version was v6.14.4.  It did work successfully on v10.15.0.

https://flaviocopes.com/package-json/#engines

Reply:
Good call, I will copy over what we are using in our other repos